src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -961,6 +961,77 @@ public static async ValueTask<FormulaValue> AsType(EvalVisitor runner, EvalVisit
             {
                 return new ErrorValue(irContext, e.ExpressionError);
             }
+        }
+
+        public static async ValueTask<FormulaValue> SearchImpl(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            var source = args[0];
+            var textToSearchArg = args[1];
+
+            if (source is BlankValue || source is ErrorValue || textToSearchArg is BlankValue)

```

Comment:
>source is ErrorValue [](http://example.com/codeflow?start=40&length=20)

This should not be true, as the wrapper would return the error if the argument is an error.